SwSpotify/__init__.py

```diff
@@ -9,9 +9,14 @@ class WebData:
 
     @staticmethod
     def set_song(data):
-        WebData.track = data['title']
-        WebData.artist = data['artist']
-        WebData.playState = data['playState']
+        try:
+            WebData.track = data['title']
+            WebData.artist = data['artist']
+            WebData.playState = data['playState']
+        except KeyError:
+            WebData.track = "Not found"
```

Comment:
I don't think it's a good idea to set them to strings because then they'll be returned as song and artists

Reply:
I see you do it to trigger SpotifyPaused but still

Reply:
> Screenshot of what it looks like?
![image](https://user-images.githubusercontent.com/25402427/73386348-ff2c2200-42ce-11ea-88e1-b3b59ff102a1.png)

Reply:
Changed that